docs/usage.md

```diff
@@ -153,7 +154,38 @@ to mark false positives in the source code. This way the suppression information
 line of code. Although it is possible, it is not recommended to suppress false positives
 on the Web UI only, because this way the suppression will be stored in a database that is unrelated to the source code.
 
-## <a name="step-3"></a> Step 3: Store analysis results in a CodeChecker DB and visualize results
+## <a name="step-3"></a>Step 3: View analysis results in command line or generate static HTML files
+You can print detailed results (including the control flow) in command line by running:
+```
+CodeChecker parse --print-steps ./reports
+...
+Found no defects while analyzing grid-view.c
+[MEDIUM] /home/ednikru/work/codechecker/play/tmux/log.c:89:1: Opened File never closed. Potential Resource leak [alpha.unix.Stream]
+}
+^
+  Report hash: 88d734fc6eeb71dd292863f2674c370a
+  Steps:
+    1, log.c:80:6: Assuming 'log_level' is equal to 0
+    2, log.c:89:1: Opened File never closed. Potential Resource leak
+
+Found 1 defect(s) while analyzing log.c
+...
+```
+
+It is possible to generate reports as plain `HTML` files using the `CodeChecker parse` command.
+```
+CodeChecker parse ./reports -e html -o ./reports_html
+...
+To view the results in a browser run:
+> firefox ./reports_html/index.html
+```
+
+`./reports_html` directory will contain an `index.html` with a link to all findings that are stored in separate `HTML` files (one per
+analyzed build action).
+
+![Analysis results in static HTML files](/docs/images/static_html.png)
```

Comment:
Above you are talking about `index.html` file but this screenshot was made from a separate HTML file.